### PR TITLE
Translations templates and slovene translations, project issue service desk reminder

### PR DESCRIPTION
**Pull request by sysadminmatmoz**
Just some translations and templates. Regarding the module, you can dismiss it or change the author, it's just a reminder that I needed.

### CODE REVIEW

File: reminder_issue_deadline/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    'name': "Reminders and Agenda for Issues",
+    'version': '1.0.0',
+    'author': 'Matmoz d.o.o.',
+    'category': 'Custom',
+    'website': 'https://www.matmoz.si',
+    # 'price': 6.00,
+    'currency': 'EUR',
+    'depends': ['reminder_base', 'service_desk_issue'],
```
Comment (yelizariev) [inline reminder_issue_deadline/__openerp__.py:9]: service_desk_issue can be replaces to built-in project_issue as service_desk_issue is not used directly

File: reminder_issue_deadline/static/description/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Reminders and Agenda for Tasks</h2>
```
Comment (yelizariev) [inline reminder_issue_deadline/static/description/index.html]: Please, change it to 

```
Reminders and Agenda for Issues
```

File: reminder_issue_deadline/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        </div>
+
+        <div class="oe_span12">
+            <div class="oe_demo oe_picture oe_screenshot">
+                <img src="form.png"/>
```
Comment (yelizariev) [inline reminder_issue_deadline/static/description/index.html:9]: Could you prepare screenshot? It should have aspect ratio 752:400

It's ok, if you skip this. I will do it myself.

**Comment by yelizariev** [comment]
Great! Thanks.
Could you please make small fixes that I have mentioned in source comments

File: reminder_issue_deadline/__openerp__.py (hunk 1 above)
Comment (sysadminmatmoz) [inline reminder_issue_deadline/__openerp__.py:9]: service_desk_issue defines the issue deadline, no deadlines on the built-in project_issue

Comment (yelizariev) [inline reminder_issue_deadline/__openerp__.py:9]: I don't see deadline field https://github.com/OCA/project-service/blob/8.0/service_desk_issue/project.py

https://github.com/odoo/odoo/blob/8.0/addons/project_issue/project_issue.py#L222

Comment (sysadminmatmoz) [inline reminder_issue_deadline/__openerp__.py:9]: The definition is done in project.issue (https://github.com/OCA/OCB/blob/8.0/addons/project_issue/project_issue.py), but the field is made available in service_desk_issue (https://github.com/OCA/project-service/blob/8.0/service_desk_issue/service_desk_view.xml)).

I could have made a new view directly in the module, but I was lazy :smile:

Comment (yelizariev) [inline reminder_issue_deadline/__openerp__.py:9]: ok, I got it

**Pull request merged**